tests/monitoring/vm_monitoring.go

```diff
@@ -40,6 +41,7 @@ import (
 	"kubevirt.io/client-go/kubecli"
 
 	"kubevirt.io/kubevirt/pkg/libvmi"
+	virtcontroller "kubevirt.io/kubevirt/pkg/monitoring/metrics/virt-controller"
```

Comment:
@machadovilaca I think its better not to use product code in e2e, but implement the needed function in-place. Otherwise you are losing the protection when refactoring the product code.

Reply:
The alternative would be to hard code the bucket values, which would still mean the need to change when refactoring or even changing those values

Reply:
@machadovilaca And this is good IMO, these are the buckets you expect to see, so its fine to hardcode them in the test, so that if someone will change them the test will fail.